templates/facets-item-list--dropdown--localgov-directories-facets.html.twig

```diff
@@ -0,0 +1,45 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a facets item dropdown.
```

Comment:
Any chance the accuracy of the above comment line can be improved?

Reply:
I think this is actually pretty consistent with the default facets templates. 
https://git.drupalcode.org/project/facets/-/blob/2.0.x/templates/facets-item-list.html.twig#L4
I couldn't think of anything better, what did you have in mind @Adnan-cds ?

Reply:
It's the LocalGov flavour of the facets dropdown rather than the default one, isn't it?

Reply:
I've updated the comment to make it clear it refers to directory facets